documentation/en/api-v0-methods.md

```diff
@@ -5875,12 +5875,12 @@ Response:
     "InitialPledge": "0",
     "ExpectedDayReward": "0",
     "ExpectedStoragePledge": "0",
-    "ReplacedSectorAge": 10101,
+    "PowerBaseEpoch": 10101,
```

Comment:
Is this not going to be an issue for API stability? And/or historical state?

Reply:
I'm honestly not sure? But it _is_ a breaking change that we've had in the protocol itself, so I feel like the API needs to break too here.

Reply:
Yeah, it does need to change. I guess: do it and see if anyone cares?